pbiapi/pbiapi.py

```diff
@@ -394,6 +394,20 @@ def take_over_dataset(self, workspace_name: str, dataset_name: str) -> None:
             logging.error(f"Takeover of dataset {dataset_name} failed!")
             self.force_raise_http_error(response)
 
+    @check_token
+    def get_dataset_refresh_history(self, workspace_name: str, dataset_name: str, top=10) -> List:
+        workspace_id, dataset_id = self.get_workspace_and_dataset_id(workspace_name, dataset_name)
+
+        url = self.base_url + f"groups/{workspace_id}/datasets/{dataset_id}/refreshes?$top={top}"
+
+        response = requests.post(url, headers=self.headers)
```

Comment:
if that one raises, you could be failing :)

Reply:
Will add this as an issue to be resolved across all methods